src/Tizen.NUI/src/internal/WebView/WebContext.cs

```diff
@@ -168,11 +168,11 @@ public bool DisableCache
         {
             get
             {
-                return disableCache;
+                return cacheDisabled;
```

Comment:
Shouldn't we have to change property name too?

Reply:
this property would be changed by another patch [1].
after [1] merged, I will update current patch.

[1] https://github.com/Samsung/TizenFX/pull/2940.

Reply:
OK. I merged #2940 just now. It uses `CachedEnabled`. Please update the patch.

Reply:
Done.